src/streamlink/plugins/dogan.py

```diff
@@ -5,6 +5,7 @@
 from streamlink.plugin import Plugin
 from streamlink.plugin.api import validate
 from streamlink.stream import HLSStream
+from streamlink.utils import parse_json
```

Comment:
Initial `validate.transform(parse_json)` calls are not needed in schemas if you use `data = self.session.http.json(res, schema=...)`
https://github.com/streamlink/streamlink/blob/9f93291294ac908af1548faf3c5823b9d6488347/src/streamlink/plugin/api/http_session.py#L100-L106